wweval/R/eval_fit.R

```diff
@@ -36,46 +43,21 @@ eval_fit_ww <- function(config_index,
   ) |> paste0(".rds")
 
 
-
-
-  # Get the evaluation data from the specified evaluation date ----------------
-  eval_hosp_data <- get_input_hosp_data(
-    forecast_date_i = eval_config$eval_date,
-    location_i = location,
-    hosp_data_dir = eval_config$hosp_data_dir,
-    calibration_time = 365 # Grab sufficient data for eval
-  )
-
-  save_object("eval_hosp_data", output_file_suffix)
-
-  eval_ww_data <- get_input_ww_data(
-    forecast_date_i = eval_config$eval_date,
-    location_i = location,
-    scenario_i = "status_quo",
-    scenario_dir = eval_config$scenario_dir,
-    ww_data_dir = eval_config$ww_data_dir,
-    calibration_time = 365, # Grab sufficient data for eval
-    last_hosp_data_date = eval_config$eval_date,
-    ww_data_mapping = eval_config$ww_data_mapping
-  )
-
-  save_object("eval_ww_data", output_file_suffix)
-
-  # Get the table of hospital admissions outliers ----------------------------
   table_of_exclusions <- tibble::as_tibble(eval_config$table_of_exclusions)
 
   # Wastewater model fitting loop-----------------------------------------------
 
-  stan_model_path_target <- get_model_path(
-    model_type = "ww",
-    stan_models_dir = eval_config$stan_models_dir
-  )
   raw_input_hosp_data <- get_input_hosp_data(
     forecast_date_i = forecast_date,
     location_i = location,
     hosp_data_dir = eval_config$hosp_data_dir,
     calibration_time = eval_config$calibration_time
-  )
+  ) |>
+    dplyr::mutate(
+      forecast_date = forecast_date
```

Comment:
I think this is meant to be 
```suggestion
      forecast_date = !!forecast_date
```